research/src/pages/file/file.proposal.mdx

```diff
@@ -0,0 +1,207 @@
+---
+menu: Components
```

Comment:
This should be proposals - although I do have a desire to move this to a different name in the future. However let's not change it in this PR.